packages/design-tokens/tokens/semantic/colour.json

```diff
@@ -20,6 +20,10 @@
       "line": {
         "value": "{base.neutral.200}",
         "type": "color"
+      },
+      "dot": {
+        "value": "{base.neutral.300}",
+        "type": "color"
```

Comment:
@thewbuk We don't directly change this file's value, this should be added by designed, if you can not find the color's semantic token, you could use hex color code directly